pkg/client/supplier/client_test.go

```diff
@@ -76,7 +76,7 @@ func TestSupplierClient_CreateClaim(t *testing.T) {
 	var (
 		signAndBroadcastDelay = 50 * time.Millisecond
 		doneCh                = make(chan struct{}, 1)
-		ctx                   = context.Background()
+		ctx                   = polyzero.NewLogger().WithContext(context.Background())
```

Comment:
This shouldn't be strictly necessary as `polylog.Ctx()` will use the polyzero logger by default if one has not been provided to the context. IIRC other  tests do not set up loggers but use this default instead.